Firestore/src/V1/Gapic/FirestoreGapicClient.php

```diff
@@ -1091,99 +894,94 @@ public function listen(array $optionalArgs = [])
+     *           returned may be fewer.
+     *
+     *           For example, this may be set to one fewer than the number of parallel
+     *           queries to be run, or in running a data pipeline job, one fewer than the
+     *           number of workers or compute instances available.
      *     @type string $pageToken
-     *          The `next_page_token` value returned from a previous call to
-     *          PartitionQuery that may be used to get an additional set of results.
-     *          There are no ordering guarantees between sets of results. Thus, using
-     *          multiple sets of results will require merging the different result sets.
-     *
-     *          For example, two subsequent calls using a page_token may return:
-     *
-     *           * cursor B, cursor M, cursor Q
-     *           * cursor A, cursor U, cursor W
-     *
-     *          To obtain a complete result set ordered with respect to the results of the
-     *          query supplied to PartitionQuery, the results sets should be merged:
-     *          cursor A, cursor B, cursor M, cursor Q, cursor U, cursor W
+     *           A page token is used to specify a page of values to be returned.
+     *           If no page token is specified (the default), the first page
+     *           of values will be returned. Any page token used here must have
+     *           been generated by a previous call to the API.
      *     @type int $pageSize
-     *          The maximum number of partitions to return in this call, subject to
-     *          `partition_count`.
-     *
-     *          For example, if `partition_count` = 10 and `page_size` = 8, the first call
-     *          to PartitionQuery will return up to 8 partitions and a `next_page_token`
-     *          if more results exist. A second call to PartitionQuery will return up to
-     *          2 partitions, to complete the total of 10 specified in `partition_count`.
+     *           The maximum number of resources contained in the underlying API
+     *           response. The API may return fewer values in a page, even if
+     *           there are additional values to be retrieved.
      *     @type RetrySettings|array $retrySettings
-     *          Retry settings to use for this call. Can be a
-     *          {@see Google\ApiCore\RetrySettings} object, or an associative array
-     *          of retry settings parameters. See the documentation on
-     *          {@see Google\ApiCore\RetrySettings} for example usage.
+     *           Retry settings to use for this call. Can be a
+     *           {@see Google\ApiCore\RetrySettings} object, or an associative array of retry
+     *           settings parameters. See the documentation on
+     *           {@see Google\ApiCore\RetrySettings} for example usage.
      * }
      *
-     * @return \Google\Cloud\Firestore\V1\PartitionQueryResponse
+     * @return \Google\ApiCore\PagedListResponse
```

Comment:
This is a BC breaking change that we need to address. It looks like `partitionQuery` became a paged response.

Reply:
Addressed in https://github.com/googleapis/google-cloud-php/pull/5096